modules/graphics/src/mrb_vertex_buffer.cxx

```diff
@@ -95,6 +99,50 @@ vbo_index_count(mrb_state *mrb, mrb_value self)
   return mrb_fixnum_value(mmrb_vertex_buffer_ptr(mrb, self)->GetIndexCount());
 }
 
+static mrb_value
+vbo_add_quad(mrb_state *mrb, mrb_value self)
+{
+  Moon::IntRect quad_rect;
+  Moon::FloatRect quad_texture_rect;
+  Moon::Vector4 color;
+  Moon::VertexBuffer *vbo = NULL;
+  Moon::Vertex vertices[4];
+  GLuint indices[] = { 0, 1, 3, 2, 3, 1 };
+
+  mrb_value rect_obj;
+  mrb_value tex_rect_obj;
+  mrb_value color_obj;
+  mrb_int i = 0;
```

Comment:
What's the deal with this? Doesn't seem to be used in any function, you can just use a static number for each vertex.